### ISSUE TITLE
sage-8.1-exec.patch fails in building 8.2.beta4

### ISSUE THREAD

**Issue by strogdon** [cschwan/sage-on-gentoo#498]
This is in Prefix. I haven't moved to Gentoo yet.
```
>>> Preparing source in /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src ...
 ^[[32;01m*^[[0m Will copy sources from /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src
 ^[[32;01m*^[[0m python2_7: copying to /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python2_7
 ^[[32;01m*^[[0m python2_7: running distutils-r1_run_phase python_prepare
 ^[[32;01m*^[[0m Adjusting to prefix /storage/strogdon/gentoo-rap
 ^[[32;01m*^[[0m   sage-env ...
^[[A^[[72C ^[[34;01m[ ^[[32;01mok^[[34;01m ]^[[0m
 ^[[32;01m*^[[0m Applying sage-8.1-sage-cython.patch ...
^[[A^[[72C ^[[34;01m[ ^[[32;01mok^[[34;01m ]^[[0m
 ^[[32;01m*^[[0m Applying sage-8.1-exec.patch ...
1 out of 26 hunks FAILED -- saving rejects to file bin/sage.rej
^[[A^[[72C ^[[34;01m[ ^[[31;01m!!^[[34;01m ]^[[0m
 ^[[31;01m*^[[0m ERROR: sci-mathematics/sage-9999::sage-on-gentoo failed (prepare phase):
 ^[[31;01m*^[[0m   patch -p1  failed with /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/files/sage-8.1-exec.patch
```
This is ugly but I think readable.

**Comment by kiwifb** [comment]
OK I missed something in the last bunch of commits. I'll fix that ASAP.

**Comment by kiwifb** [comment]
Closed via 85a81147d0c07e91de2c8759b59453ae617648c8
